Formula/gojq.rb

```diff
@@ -1,22 +1,18 @@
 class Gojq < Formula
   desc "Pure Go implementation of jq"
   homepage "https://github.com/itchyny/gojq"
-  url "https://github.com/itchyny/gojq/archive/v0.12.0.tar.gz"
-  sha256 "3f280536f53ab6670b89424b0cd295d096b2d2928a5ed5d53bc3a23333051c05"
+  url "https://github.com/itchyny/gojq.git",
+      tag:      "v0.12.0",
+      revision: "203d5e1dfd435460500ea5b00f71eca48bf1cb88"
```

Comment:
Why can't we use the tarball?

Reply:
Ah, I just saw your comment. We can keep the `git` checkout to get the version information, but let's stick to  `go build` instead of `make`. See `buildkit` for an example of how to set the `ldflags` correctly.

Reply:
because we need git revision info when build . follow https://github.com/itchyny/gojq/pull/59

Reply:
Yes, I see. Please see the `buildkit` formula for the correct way to do this, and please update your commit message to the correct format.